### PR TITLE
Task/95 about page

### LINKED ISSUE #95: Create about page

### PR DESCRIPTION
**Pull request by bellangerq**
- Create about page.
- Add content.
- Create External helper component.

Closes #95.

### CODE REVIEW

**Comment by ThunderDev1** [comment]
When navigating to the about page, the scroll position seems wrong. I was expecting to be at the top of the page, but it's a bit lower than that 🤔

**Comment by ThunderDev1** [comment]
We could add our avatars in the team section. What do you think ?

**Comment by ThunderDev1** [comment]
Maybe add a few technical details about the website, eg. This site was built with React.js, dotnet core, uses custom css, respects a11y guidelines, uses websocket communication...

**Comment by bellangerq** [comment]
> When navigating to the about page, the scroll position seems wrong. I was expecting to be at the top of the page, but it's a bit lower than that 🤔

Yes, this is more a global setting in the app. I agree we should disable this.

**Comment by bellangerq** [comment]
> We could add our avatars in the team section. What do you think ?

Mmh any idea?

File: client/src/pages/About/About.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            </a>
+            , réalisé par Charles Bossart et Pauline Berdal et édité par la{' '}
+            <a href='https://okaluda.fr/'>
+              société Oka Luda
+              <External />
+            </a>
+            . Merci à eux !
+          </li>
+          <li>
+            L'icône de poule a été réalisé par Freepik sur{' '}
```
Comment (ThunderDev1) [inline client/src/pages/About/About.tsx]: Typo ?

File: client/src/pages/About/About.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            <External />
+          </a>{' '}
+          et{' '}
+          <a href='https://www.linkedin.com/in/stephen-granger-907656109/'>
+            Stephen
+            <External />
+          </a>
+          . Pour toute question, suggestion d'amélioration ou remontée de bugs,
+          n'hésitez pas à nous contacter à cette adresse :{' '}
+          <a href='mailto:contact@poule-poule.com'>contact@poule-poule.com</a>.
```
Comment (hissalht) [inline client/src/pages/About/About.tsx]: Do we even own this domain ?

File: client/src/components/External/External.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@import 'styles/variables';
+
+.External {
+  fill: currentColor;
```
Comment (hissalht) [inline client/src/components/External/External.scss]: neat !

File: client/src/pages/About/About.tsx (hunk 2 above)
Comment (bellangerq) [reply client/src/pages/About/About.tsx]: Huhu, no. What do I put instead?

File: client/src/pages/About/About.tsx (hunk 1 above)
Comment (bellangerq) [reply client/src/pages/About/About.tsx]: Where?

Comment (ThunderDev1) [reply client/src/pages/About/About.tsx]: réalisé -> réalisée ? 😄

**Pull request merged**